workflow-language/src/main/java/com/symphony/bdk/workflow/swadl/v1/activity/request/ExecuteRequest.java

```diff
@@ -16,7 +16,7 @@
 @Data
 public class ExecuteRequest extends BaseActivity {
   private String url;
-  private String method;
-  private Map<String, Object> body = Collections.emptyMap();
-  private Map<String, String> headers = Collections.emptyMap();
+  private String method = "GET";
```

Comment:
should it be an enum?

Reply:
It could be an enum but I don't think we need that in java model since the json-schema verifies the value at SWADL level (it is defined as an enum in the schema).

Reply:
but using an enum makes it cleaner in the code (especially when we had the switch)
now without the switch keeping a string is probably simpler (but could be error prone)